IOIOLibAndroid/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'com.github.dcendents.android-maven'
 android {
     compileSdkVersion 30
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 21
```

Comment:
In libraries it's not mandatory, only the apks have this issue

Reply:
I will change the libraries back to 14 then.